packages/frontpage/lib/data/atproto/did.ts

```diff
@@ -1,15 +1,18 @@
 import { cache } from "react";
 import { z } from "zod";
-import { PlcDidDocumentResolver } from "@atcute/identity-resolver";
-import type { DidDocument } from "@atcute/identity";
+import {
+  CompositeDidDocumentResolver,
+  PlcDidDocumentResolver,
+  WebDidDocumentResolver,
+} from "@atcute/identity-resolver";
+import { type DidDocument, isAtprotoDid } from "@atcute/identity";
 import { FRONTPAGE_APPVIEW_USER_AGENT } from "@/lib/constants";
 
 type Brand<K, T> = K & { __brand: T };
-export type DID = Brand<`did:plc:${string}`, "DID">;
+export type DID = Brand<`did:${"plc" | "web"}:${string}`, "DID">;
```

Comment:
Now that we have the template literal type the brand has less value, but it's still nice to have because it's much easier to trace where exactly DID values are being created. Compared to just the template literal where any line of code can just do

```ts
`did:web:example.com` as const
```

And that is now a value assignable to something that accepts a DID.